## 📋 Quality Gate Results

| Check | Status |
|-------|--------|
| Knip | ✅ success |
| Lint | ❌ failure |
| Slop | ✅ success |
| Build | ✅ success |
| Infra Tests | ❌ skipped |
| Unit Tests | ❌ skipped |
| Component Tests | ❌ skipped |
| Perf Tests | ❌ skipped |
| Visual Tests | ❌ skipped |


### ❌ Lint Failure Details
```text

> hrm@0.30.0 lint /home/runner/work/hrm/hrm
> eslint app/ components/ constants/ context/ hooks/ lib/ services/ tests/ types/ utils/ server.ts proxy.ts --cache


/home/runner/work/hrm/hrm/tests/unit/webSocketReducer.test.ts
  24:27  error  Replace `overrides:·Partial<TestHrmData>·=·{}` with `⏎····overrides:·Partial<TestHrmData>·=·{}⏎··`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

 ELIFECYCLE  Command failed with exit code 1.
```
### ❌ Infrastructure Test Failure Details
```text
Log file not found.
```
### ❌ Unit Test Failure Details
```text
Log file not found.
```
### ❌ Component Test Failure Details
```text
Log file not found.
```
### ❌ Visual Test Failure Details
```text
Log file not found.
```
### ❌ Performance Test Failure Details
```text
Log file not found.
```
⚠️ **Some checks failed.** Full logs available in [workflow artifacts](https://github.com/arii/hrm/actions/runs/21773518601).

---
> Report generated for commit: `cdadd8418a55d0e18057e48f97769601db7b335a`

--- a/tests/unit/webSocketReducer.test.ts
+++ b/tests/unit/webSocketReducer.test.ts
@@ -21,7 +21,9 @@ interface TestHrmData extends HrmData {
 describe('webSocketReducer', () => {
   // Helper to create mock users and reduce duplication
   // Returns TestHrmData to allow for extra test properties
-  const createMockUser = (overrides: Partial<TestHrmData> = {}): TestHrmData => ({
+  const createMockUser = (
+    overrides: Partial<TestHrmData> = {}
+  ): TestHrmData => ({
     clientId: 'client-1',
     name: 'User A',
     age: 30,

